rusk-schema/Cargo.toml

```diff
@@ -3,6 +3,16 @@ name = "rusk-schema"
 version = "0.1.0"
 edition = "2021"
 
+license = "MPL-2.0"
+readme = "README.md"
+
+description = "Schema definition for all Rusk GRPC services"
+keywords = ["protobuf", "grpc", "schema"]
```

Comment:
Add `rusk` maybe?

Reply:
I prefer to be generic and unfortunately they took `rusk` on crates.io (to be fair, we weren't going to publish there anyway), so I want to avoid confusion.